tests/phpunit/tests/test-links-directory.php

```diff
@@ -202,10 +211,30 @@ public function test_get_language_from_url_with_wrong_ssl() {
 	/**
 	 * @ticket #1296
 	 * @see https://github.com/polylang/polylang/issues/1296.
+	 *
+	 * @param bool $cache_languages Value of the constant `PLL_CACHE_LANGUAGES`.
+	 * @param bool $cache_home_url  Value of the constant `PLL_CACHE_HOME_URL`.
+	 *
+	 * @testWith [true, false]
+	 *           [false, true]
+	 *           [false, false]
 	 */
-	public function test_flag_url_with_subfolder_install() {
+	public function test_flag_url_with_subfolder_install( $cache_languages, $cache_home_url ) {
 		$this->maybe_set_subfolder_install( true );
 
+		Functions\when( 'pll_get_constant' )->alias(
+			function ( $constant_name ) use ( $cache_languages, $cache_home_url ) {
+				switch ( $constant_name ) {
+					case 'PLL_CACHE_LANGUAGES':
+						return $cache_languages;
+					case 'PLL_CACHE_HOME_URL':
+						return $cache_home_url;
+					default:
+						return null;
```

Comment:
Shouldn't default be?
```php
return constant( $constant_name );
```
Otherwise if the function is called during the code execution for something different than `PLL_CACHE_LANGUAGES` or `PLL_CACHE_HOME_URL` it might return a wrong value, am I right?